lib/service/service.go

```diff
@@ -346,7 +348,25 @@ func (process *TeleportProcess) initAuthService(authority auth.Authority) error
 
 	// Heart beat auth server presence, this is not the best place for this
 	// logic, consolidate it into auth package later
-	var authClient *auth.TunClient
+	storage := utils.NewFileAddrStorage(
```

Comment:
there's `TeleportProcess.connectToAuthServer()` which does exactly what this block of code does.